bb-worker/Dockerfile.debian8-build-python3.7

```diff
@@ -0,0 +1,49 @@
+FROM debian:8
+ARG DEBIAN_FRONTEND=noninteractive
+ARG DUMBINIT_VERSION=v1.2.2
```

Comment:
DUMBINIT_VERSION is set here but not used

Reply:
Removed!